### DISCUSSION TITLE
v0.7.0

### DISCUSSION THREAD

**Discussion by af** [af/hyperpaper-planner#13] [Announcements]
<hr /><em>This discussion was created from the release <a href='https://github.com/af/planner-pdf/releases/tag/v0.7.0'>v0.7.0</a>.</em>

**Comment by gregor1991** [comment]
Very nice update! Somehow I really like these weekly overview, while I'm not 100% sure, how usefull they are, since I have to write down/reference the date, I'm planning for (I would use them as kind of weekly overview). But on the other hand, they are very flexibel and I like that. Nevertheless, I would prefer when they are at the beginning of each week (I guess it is extremely complicated/time consuming, right?). 
The other changes I really appreciate, so the additional links, the minie calendar etc.. Will be very usefull!
The holidays: In general they would be great. But I guess, it will be a lot of work to implement holidays for a lot of country. It is crazy, but in Germany, in almost every strate, we have different holidays :D. They don't differ extremely, but in some days at least.

**Reply by af** [reply]
I'm not sure if I can accommodate every German state 😬 but support for a single German holiday list is certainly doable. Let me know if you know of a good canonical reference for that.

Just sent you an email about the weekly page order question :)

edit: the email to you bounced back, could you submit the survey form again and double check your address?

**Reply by gregor1991** [reply]
Regarding the german stuff: For me you absolutely don't have to. But to be honest: it is hard to keep track of it, even if you are living in Germany :D.
As far as I know, the standard holidays are these, according to the state https://www.bmi.bund.de/EN/topics/constitution/national-holidays/national-holidays.html - I only added the dates and some explanation:
- New Year's holiday: 1st January
- Good Friday: 15.04.2022
- Easter Monday: 18.04.2022 
- 01.05.2022 - the official name might be 1. Mai, but also quite common: Labour day or International Workers' Day
- Ascension Day: 26.05.2022
- Pentecost Monday: 06.06.2022
- Day of German Unity: 03.10.2022
- Christmas is a bit complicated: We celebrate Christmas on the 24th (no holiday) (presents etc.), but on the 25th and 26th there are holidays. I only found Christmas day and Boxing day as translation, but I think this is also caused by the different traditions. So I'm not sure, which would be the perfect names there. Maybe just Christmas holiday or Christmas feast day or so? I don't know.


Regarding the mail: I'm stupid. I'm pretty sure, that I added a point, where no point should be. So directly after the first letter. If it is easier for you, that I fill it out again, please let me know. I found the link to the survey.

**Reply by af** [reply]
Thanks! Just re-sent the email 🤞 Will add those holidays into the system 👍
